Python/Product/PythonTools/PythonTools/Repl/Completion/ReplRemoteCompletionBroker.cs

```diff
@@ -146,11 +148,15 @@ CancellationToken token
             // this code here is just good enough to test simple cases
             // in old codebase, we used to get the expression from the analyzer
             var expression = triggerPoint.Snapshot.GetText();
-            if (expression.EndsWith(".")) {
-                expression = expression.Substring(0, expression.Length - 1);
+
+            var finder = new ExpressionFinder(expression, PythonLanguageVersion.V37, FindExpressionOptions.Complete);
```

Comment:
Language version should not be hard coded. There are couple of ways you could obtain it:

1. Cast `eval` in the calling function `as` a `PythonCommonInteractiveEvaluator`, and look at `LanguageVersion` property on that. Cast will be null for debug repl though, might be best to add the property to `IPythonInteractiveIntellisense` then we can make it accurate for all and no need to cast.

2. Use the factory available on language client: `client.Factory.Configuration.Version.LanguageVersion`